docs/docusaurus.config.js

```
@@ -22,7 +22,7 @@ const config = {
   organizationName: 'objectiv', // Usually your GitHub org/user name.
   projectName: 'objectiv.io', // Usually your repo name.
 
-  onBrokenLinks: 'log',
+  onBrokenLinks: 'throw',
   onBrokenMarkdownLinks: 'throw',
```

Only functional change in the whole branch. I'm not sure why was this set to log, it's not like we are gonna deploy broken links... are we? :)

---

Log can be useful to collect all broken links at once, but I don't think we need that anymore?

@borft FYI

---

Good point, maybe we can make this variable dynamic depending on the env then.

We can leave it as `log` in dev and make into `throw` while building for prod

---

That sounds good. WDYT @borft

---

I tested this real quick and the throw also builds a report of all broken links, which is printed to the console before aborting the build.

The difference seems to be that log allows you to click around anyway. Eg: you can dismiss the crashes.
The throw will not allow yarn start or yarn build to complete instead.

---

Well, I don't mind the build/env breaking on broken links actually ;) This would prevent you from committing broken stuff (unless you commit untested stuff). 

So I'd be OK with throw, rather than log

---

Cool 👌🏼  

If anyone needs to do a heavy refactoring, they can always temporarily change this option on their local env of course.